geode-core/src/main/java/org/apache/geode/internal/cache/DiskStoreImpl.java

```diff
@@ -695,7 +695,7 @@ void put(InternalRegion region, DiskEntry entry, ValueWrapper value, boolean asy
       throws RegionClearedException {
     DiskRegion dr = region.getDiskRegion();
     DiskId id = entry.getDiskId();
-    long start = async ? this.stats.startFlush() : this.stats.startWrite();
+    long start = async ? this.getStats().startFlush() : this.getStats().startWrite();
```

Comment:
I really like that you changed some of these field accesses to calling a gettor.
I think IntelliJ make it easy for you to get rid of all these "this." prefixes on what is now method calls.
If you can try to clean these up but it could be cleaned up later